packages/node_modules/@webex/react-component-loading-screen/src/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import Spinner from '@ciscospark/react-component-spinner';
+
+import styles from './styles.css';
+
+const propTypes = {
+  loadingMessage: PropTypes.string
+};
+
+const defaultProps = {
+  loadingMessage: ''
+};
+
+function LoadingScreen({loadingMessage}) {
+  return (
+    <div className={classNames('ciscospark-loading', styles.loading)}>
+      <div className={classNames('ciscospark-spark-logo', styles.logo)} />
```

Comment:
Should we rename classnames while we're migrating things over? I feel like if we don't do it now, it won't get done anytime soon.

Reply:
Yea, good catch. `webex-teams` a good prefix?

Reply:
Fixed

Reply:
I may be missing something, but I still see `ciscospark` for class names instead of `webex-teams`.

Reply:
Oh great, I fixed the wrong component... Stand by